crates/sui-core/src/consensus_adapter.rs

```diff
@@ -189,6 +190,12 @@ pub trait CheckConnection: Send + Sync {
     fn update_mapping_for_epoch(&self, authority_names_to_peer_ids: HashMap<AuthorityName, PeerId>);
 }
 
+pub trait CheckReputationScore: Send + Sync {
+    fn get_low_scoring_authorities(&self) -> DashMap<AuthorityName, u64>;
+
+    fn update_mapping_for_epoch(&self, low_scoring_authorities: Arc<DashMap<AuthorityName, u64>>);
+}
```

Comment:
This seems to be unused - I believe it's not needed given the current design, right?